tests/test_adapters.py

```diff
@@ -38,12 +38,11 @@ def test__indent(self) -> None:
 
     def test_step(self, caplog) -> None:
         manager = StdOutManager()
-        test_message = "Wow. I’m Mr. Manager."
+        test_message = "Wow. I'm Mr. Manager."
```

Comment:
I noticed this one too.  I'm really curious how the other character ended up there?

Reply:
Mac, by default, will put the smartquotes in, especially on copy/paste. I must have either copied the quote from a site that had the smartquote there or had written it in here before i angrily turned off the setting that makes smartquotes everywhere. 😩